guvnor-services/guvnor-services-backend/src/main/java/org/guvnor/common/services/backend/metadata/attribute/GeneratedFileAttributes.java

```diff
@@ -0,0 +1,11 @@
+package org.guvnor.common.services.backend.metadata.attribute;
+
+import org.uberfire.java.nio.file.attribute.BasicFileAttributes;
+
+/**
+ * Represents attributes of a file which has been generated.
+ */
+public interface GeneratedFileAttributes extends BasicFileAttributes {
```

Comment:
@manstis Unfortunately UF currently imposes extending of `AbstractBasicFileAttributeView` https://github.com/uberfire/uberfire/blob/master/uberfire-io/src/main/java/org/uberfire/io/impl/IOServiceDotFileImpl.java#L188 in order for attributes to be loaded, therefore the hierarchy cannot be avoided (or at least I missed a way to do so!).